### DISCUSSION TITLE
FP 1.2.12_14 new feature explanation: Preferences…->Behaviour->Link to emails from text

### DISCUSSION THREAD

**Discussion by Illegit** [freeplane/freeplane#1945] [Help] [answered]
Hi, can you explain how it works? 
Preferences…->Behaviour->Link to emails from text
Are these as links to single emails? Or to email accounts?

Thanks

**Comment by dpolivaev** [comment, marked as answer]
See https://github.com/freeplane/freeplane/issues/1861
I have just changed it to "Link to emails from pasted text"

**Comment by Illegit** [comment]
Thank you. Clear.
